Neos.ContentGraph.DoctrineDbalAdapter/Tests/Behavior/Features/Projection/ProjectionIntegrityViolationDetection/SubtreeTagsAreInherited.feature

```diff
@@ -58,18 +58,20 @@ Feature: Run integrity violation detection regarding restriction relations
       | parentNodeAggregateId       | "sir-nodeward-nodington-iii"                             |
       | nodeName                    | "child-document"                                         |
       | nodeAggregateClassification | "regular"                                                |
-    And the event NodeAggregateWasDisabled was published with payload:
+    And the event SubtreeWasTagged was published with payload:
```

Comment:
Reminder: The `DisableNodeAggregate` and `EnableNodeAggregate` commands are still supported but they will internally lead to `SubtreeWasTagged` and `SubtreeWasUntagged` events now